ksp_plugin/pile_up.hpp

```diff
@@ -194,6 +197,10 @@ class PileUp {
   // good as Hari Seldon we only do it over a short period of time.
   DiscreteTrajectory<Barycentric>* psychohistory_ = nullptr;
 
+  // The angular momentum of the pile up with respect to its centre of mass.
+  // TODO(egg): This is not yet serialized.
```

Comment:
It doesn't seem hard to serialize.  Could you do it in this PR?

Reply:
I don't want to do that yet: until we use it to perform a correction, it will go completely out of synch with the game every time we warp.